.travis.yml

```diff
@@ -12,6 +12,14 @@ install:
 - npm run build
 script:
 - npm test --silent
+deploy:
+  provider: npm
+  email: dev@xcomponent.com
+  api_key:
+    secure: EO9OP5OMhrER1hV9kcfz5SM+b1l/NXOo18EfxLaCxK2tQ4rDMODkhonyaO/ULxvPgOD1F1dHYKLGnjC03Xhxyh+aHUG/nl5AoxL9zMiieVy4TXI+Lu3jnRBnCnFJfKKRyB4fd8jjHYVxdI4NQV97jrWQ+aXiSo5fJFZe0A27iKXUoKeqo+T3pTvKzFPzqj0Lsr4g7KRw0FBuEnxq6ZTtcREWSxz6UUjw8u+CnDkGFb3Ol4U9RQO1DFPVQl0eAnLeTdfcVeHy6HYu6NTqQpmDhetj8L6QfO5w/FMFUr4YindLN012748u79IYzvb400HrhyZQd5RRc8WklT0kJ2hFtvbBtO5h698PG+x+GA0Tlkls194nH6Ogm8OyKQx9J+MUNUfpSFYpABfbrp+JMAXTGrU1+PhvLy5RT3vPufqgTypG4P0R73DC2UP4n0/38uqVVVyu6TB7b2KIUWfPFFcEW4eaTZJnN7EDnI3FVfIesgvHrgw+LZ7bT+LRwm1F7b2tdCCNTABtR9QlLsGjLoPXseBIpBfEwxh4j5OlUldgreJKR/mGULdq2ZEbyDK864d1GgDhjrtC4AiN0B0KKAt6xQdaN3NAGIuqiF+rZ3PScRHUePca1qBXDQHw0fcM4TrynYHqmMMm9yDigDqXb9ojXofXnhta94aowzxrAURoJn0=
+  on:
+    branch: master
+    condition: "`./checkTag.sh` = OK"
```

Comment:
And you forgot to add the checkTag.sh file...

Reply:
resolve #96

Reply:
it is not not #96 but #95 ;-)